src/typings/prop-types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'prop-types' {
+    import React = require('react');
+
+    export const bool: {
+        isRequired: React.Validator<any>
```

Comment:
Since the whole project is kind of huge, I didn't wish to load everything for this.
Is the way I did it in my second commit acceptable?